### ISSUE TITLE
Redirect history

### ISSUE THREAD

**Issue by stevenvachon** [jhiesey/stream-http#72]
Is it not accessible via the use of `fetch`'s `{redirect:'manual'}`?

**Comment by jhiesey** [comment]
Yes, this should be possible. I will look into adding this feature in the near future.

**Comment by jhiesey** [comment]
OK, so reading [the fetch spec](https://fetch.spec.whatwg.org/#concept-filtered-response-opaque-redirect) it looks like responses corresponding to redirects have type `"opaqueredirect"`, which hides the status code, message, headers, and body.

This will break things that actually expect these things in node, like [`request`'s redirection logic](https://github.com/request/request/blob/master/lib/redirect.js). So just adding support for manual redirects won't really do the right thing.

What's the right thing to do? Should `stream-http` give redirects a fake 3xx status code?
